release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -2333,3 +2333,213 @@ To upgrade an existing {product-title} 3.10 or 3.11 cluster to this latest
+takes less time to complete.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1682924[*BZ#1682924*])
+
+* The `oreg_url` variable would not function correctly on disconnected installs
+using Satellite because the etcd image could not perform pulls on disconnected
+installs. Now, guidance and examples have been added to specify the etcd image
+URL isuing `osm_etcd_image` in the associated documentation.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1689796[*BZ#1689796*])
+
+* If a build pod was evicted, the build reported a `GenericBuildFailure`.
+Determining the cause of build failures was difficult as a result. Now a new
+failure reason, `BuildPodEvicted`, has been added.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1690066[*BZ#1690066*])
+
+* Nodes would sometimes panic due to cadvisor index reporting out of range errors.
+This has now been resolved by a backporting of kube code.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1691023[*BZ#1691023*])
+
+* ElasticSearch could not be monitored with Prometheus because the `oauth-proxy`
+was not passing a user's token. Now, the token is exchanged to ElasticSearch and
+users with proper roles can retrieve metrics in Prometheus.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1695903[*BZ#1695903*])
+
+* Deploying nodes would fail in the `setup_dns.yaml` playbook during multi-node
+setup. This was resolved by fixing the hostname that was passed to the
+`add_host` function. Now, multi-node setup proceeds as expected.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1698922[*BZ#1698922*])
+
+* Upgrading beteween minor versions would fail because several OpenShift variables
+were not used during the upgrade process. Now, `api_port` and other
+apiserver-related variables are read during the upgrade process and upgrades
+complete successfully.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1699696[*BZ#1699696*])
+
+* ElasticSearch would fail to start due to invalid certificate dates if hosts had
+non-UTC timezones. When OpenShift Nodes's timezone is not set to UTC, the
+current non-UTC timestamp is used for the `NotBefore` checking. If the timezone
+is ahead of UTC, the `NotBefore` checking would fail. Now, regardless of the
+nodes' timezone, the UTC timestamp is set to the start date in the certificates
+and failures are not reported due to non-UTC timestamps.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1702544[*BZ#1702544*])
+
+* `CustomResourceDefinition` errors were presented in a confusing manner that made
+troubleshooting difficult. Now, the CRD error messages have been clarified to
+assist in troubleshooting CRD errors.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1702693[*BZ#1702693*])
+
+* There was a missing `@` for an instance variable in the Fluentd remote
+syslog plugin code. In some cases, systemd-journald logged errant values.
+This resulted in rsyslog forwarding failures. Now, the variable has been
+corrected remote logging completes successfully.
```

Comment:
Missing word here in "Now, the variable has been corrected remote logging completes successfully.". Should there be an "and" between "corrected" and "remote"?

Reply:
Yep, added. Thanks!